### DISCUSSION TITLE
"Zooz 800 Series Z-Wave Long Range USB Stick ZST39 LR" and "Zooz 800 Series Z-Wave Long Range GPIO Module ZAC93 LR"

### DISCUSSION THREAD

**Discussion by Hedda** [zwave-js/zwave-js#5615] [General]
FYI, Zooz has apparently already released several different variants of Z-Wave 800 Series USB and GPIO module Z-Wave Controller hardware.

Note! There is still an open issue tracking Z-Wave JS support for these newer Z-Wave 800 Series Controller -> https://github.com/zwave-js/node-zwave-js/issues/5257

"Zooz 800 Series Z-Wave Long Range USB Stick ZST39 LR" available in two variants; (for US/CA/MX RF region for LR, and EU/UK RF region).

* https://www.getzooz.com/zooz-zst39-z-wave-long-range-usb-stick/
  * https://www.amazon.com/Z-Wave-ZST39-Assistant-HomeSeer-Software/dp/B0BW171KP3
  * https://www.thesmartesthouse.com/products/zooz-800-series-z-wave-long-range-usb-stick-zst39

Update! FYI, I send a qustion to Zooz Support Team and got a reply from them stating that is it the same hardware with different firmware:

"**_The ZST39 US/EU/UK uses the same hardware. The only difference is the frequency in the firmware. This is different from the Aeotec 700 series stick because the 800 series chip has already integrated the internal filter. The hardware of the 800 series will be the same for all frequencies. The 700 series does not have an internal filter._** "

![image](https://user-images.githubusercontent.com/6320001/229767588-016fbe96-2224-4898-a125-4de0402916f6.png)

![image](https://user-images.githubusercontent.com/6320001/229528827-a92ea1ee-d7fd-4e33-8306-29dc1c65f685.png)

"Zooz 800 Series Z-Wave Long Range GPIO Module ZAC93 LR" (only looks to be available as a global RF region model?)

* https://www.getzooz.com/zac93-gpio-module/
  * https://www.amazon.com/Z-Wave-Module-ZAC93-Assistant-Raspberry/dp/B0BW4SRCXL
  * https://www.thesmartesthouse.com/products/zooz-800-series-z-wave-long-range-gpio-module-zac93-lr

![image](https://user-images.githubusercontent.com/6320001/229525599-8f2b5325-0ca9-4907-9a39-ee38149a6f04.png)

![image](https://user-images.githubusercontent.com/6320001/229767826-f73a2471-12cd-493d-a098-d2a7a6e2f5df.png)

Site says they tested GPIO Shield/Header Board Module using GPIO pins inside "Home Assistant Yellow" appliance for internal mounting:

* https://www.home-assistant.io/yellow/
  * https://yellow.home-assistant.io/
    * https://www.crowdsupply.com/nabu-casa/home-assistant-yellow
      * https://community.home-assistant.io/t/home-assistant-yellow/360172

![image](https://user-images.githubusercontent.com/6320001/229768269-c8b88346-3c81-4580-9f85-2dc2be2398e2.png)

![image](https://user-images.githubusercontent.com/6320001/230123054-e6f5db4e-a77a-4bb1-8e85-6ea5083a4d8d.png)

Z-Wave 800 Series LR (Long Range) ZG23 SoC and ZGM230S SiP with same chip embedded offers up to 1-mile / over 1-kilometer in ranges

https://www.silabs.com/wireless/z-wave/introduction-to-z-wave-800-series

https://www.silabs.com/wireless/z-wave/800-series-modules

**Comment by Hedda** [comment]
Note! I do suspect that the GPIO board can see negative effects from [electromagnetic interference](https://en.wikipedia.org/wiki/Electromagnetic_interference) if mount it internally next to NVMe-SSD if neither the M.2 SSD-disk nor the underside of the Z-Wave radio GPIO Shield/Header Board Module also have [electromagnetic shielding](https://en.wikipedia.org/wiki/Electromagnetic_shielding), or?

See examples:
* https://www.sparkfun.com/news/3123
* https://www.harwin.com/blog/emc-emi-shielding-explained/

![image](https://user-images.githubusercontent.com/6320001/229815050-164f18b0-982c-4d15-a22f-e3f1efb84979.png)

![image](https://user-images.githubusercontent.com/6320001/229817418-790655ce-35a9-466f-9315-888c358c2427.png)

Personally, if I would buy this specific product then I would at the very least want to add a [electromagnetic shielding](https://en.wikipedia.org/wiki/Electromagnetic_shielding) in the form of a heatsink on the top of the NVMe-SSD, as well as maybe a layer of Kapton tape and then a layer of conductive tape on the bottom side of the Z-Wave board GPIO shield/header radio adapter, as that could potentially provide some protection from EMI/RMI/EMF interference that comes underneath (e.i. from the NVMe-SSD).

See example: 

![image](https://user-images.githubusercontent.com/6320001/229804324-11f2bd9c-ce2d-4406-84bb-be4a895132d8.png)

![image](https://user-images.githubusercontent.com/6320001/229805879-7f88e02a-7b77-4a48-b5e7-e44b051f2a34.png)

PPS: If I would design such Z-Wave board to sell as a GPIO shield/header radio adapter myself then I would not only add a professionally made grounded metal-cover as [electromagnetic interference](https://en.wikipedia.org/wiki/Electromagnetic_interference) (perfectly sized to cover just the radio SoC and all the other SMD components on the board and only expose the antenna), but I would also choose to manufacture the board using a multilayer PCB with high internal layer copper thickness for internal layers simply because a grounded internal layer could add some additional protection from EMI/RMI/EMF interference.

**Comment by Turner-Home** [comment]
I have the ZAC93 LR on my Home Assistant Yellow PoE edition.   I've deployed with a Pi CM4 Lite with 4GB and a 1TB SSD nVMIE drive.

I've not had any issues yet, solid.   No dropped messages and no timed out callbacks.   Just successful messages (RX and TX).
While I've a large number of devices, I don't have any that are long range, so I cannot attest to LR support changes.

**Comment by Hedda** [comment]
FYI, people are reporting various issues migrating backup and restore from a Z-Wave 500 or 700 to a Z-Wave 800 series controller -> https://github.com/zwave-js/node-zwave-js/issues/5257

Also, see these related threads in the Home Assistant community forums with reports that restoring to 800 series will brick the controller(!):  

- https://community.home-assistant.io/t/zwave-500-to-800-migration-gocontrol-husbzb-1-zooz-zst39/627938/

- https://community.home-assistant.io/t/migrating-from-aeotec-z-stick-700-to-zooz-800-z-wave-usb-stick/560340

- https://community.home-assistant.io/t/migrating-from-zooz-700-to-zooz-800-z-wave-usb-stick/572000

Update! There appears to be a hacky workaround that some people have managed to use in order to migrate from 700 to 800 controller:

https://www.youtube.com/watch?v=lktIFOcg7GI&ab_channel=CornellWilliams

PS: There are some talks hinting that this will be probably fixed in future firmware versions (and will then first need to your old controller).

**Comment by Turner-Home** [comment]
Just wanted to follow up after more time to confirm that a HA Yellow PoE edition with the CM4 Lite 4GB with Bluetooth/WiFi, a 1TB nVME SSD without any shielding than the sticker, along with the ZAC93 LR GPIO HAT and it is working fine.
I have a large mix of both Z-Wave and Zigbee (Sonoff relays, LED controller for kitchen under counter lighting and a number of Aqara Contact sensors for windows).
I've not seen any indications of failed communications or other issues related to potential RF.
In addition, while not ideal, I've had little issues with BlueTooth control of several earlier lightbulbs that are BlueTooth controlled.   I've considered purchasing an external antenna and enabling it, but it's not been a requirement so far.

**Comment by Hedda** [comment]
FYI, Z-Wave 800 series (including LR Long Range in the US) is now fully supported in Z-Wave JS -> https://github.com/zwave-js/node-zwave-js/issues/5257

But note that Z-Wave Controller firmware versions after version 7.18.8 (from Silicon Labs Gecko SDK Version 4.1.6) and later however have a know lockup issue / deadlock bug; with Z-Wave JS UI reposting it as "**_Z-Wave Controller firmware based on Z-Wave SDK 7.19.3 have a bug that causes the controller to randomly hang during transmission until it is restarted. It is currently unclear if this bug is fixed in a later firmware version_**."

Silabs have now apparently said that they plan to have a fix for that lockup in 7.21.2 which is scheduled for mid April of 2024, but then we will have to wait util Zooz releases a custom firmware based on that version since the standard firmware from GitHub do not work on these. 

Catch-22 is that NVM Backup/Restore for Z-Wave 500 to Z-Wave 700/800 migration requires a firmware based on 7.19.0 or higher.

Regardless, even in the future it will be important to first update (downgrade) to 7.18.8 first before updating to any 7.2x.x+ version!
